src/middlewared/middlewared/plugins/apps/pydantic_utils.py

```diff
@@ -29,6 +29,19 @@ def _validate_absolute_path(value: str) -> str:
     return os.path.normpath(value.rstrip('/'))
 
 
+def _validate_host_path(value: str) -> str:
+    """
+    Validate that a host path exists
+    """
+    if value == '':
+        return value
+
+    if not os.path.exists(value):
```

Comment:
@yocalebo do we want stat() calls in the schema? This is a blocking filesystem operation.